network.c

```diff
@@ -1389,6 +1449,18 @@ static const struct iiod_client_ops network_iiod_client_ops = {
 	.read_line = network_read_line,
 };
 
+static const struct iiod_client_ops network_iiod_client_ops_no_trunc = {
+	.write = network_write_data,
+	.read = network_read_data,
+	.read_line = network_read_line_no_trunc,
+};
+
+bool msg_trunc_supported(struct iio_network_io_context *io_ctx)
+{
+	int ret = network_recv(io_ctx, NULL, 0, MSG_TRUNC);
```

Comment:
While MSG_TRUNC is defined on other platforms I think only Linux supports MSG_TRUNC as a flag to recv(). Maye we should add a 
```
#ifdef __linux__
  ...
#elif
  return false;
#endif
```
around this.